fud2/src/main.rs

```diff
@@ -117,19 +114,32 @@ fn build_driver() -> Driver {
 
         Ok(())
     });
-    bld.op(
-        "simulate",
-        &[sim_setup],
-        simulator,
-        dat,
-        |e, input, output| {
-            e.build_cmd(&["sim.log"], "sim-run", &[input, "$datadir"], &[])?;
-            e.arg("bin", input)?;
-            e.arg("args", "+NOTRACE=1")?;
-            e.build_cmd(&[output], "json-data", &["$datadir", "sim.log"], &[])?;
+    let testbench_normal_setup = bld.setup("Normal Testbench Setup", |e| {
+        // The Verilog testbench.
+        e.var("testbench", &format!("{}/tb.sv", e.config_val("rsrc")?))?;
+        Ok(())
+    });
+    let testbench_mem_externalized_setup =
```

Comment:
Here is a suggestion purely about naming stuff: what if we came up with a single, short, made-up word that refers to this "externalized memory" concept? Even if it's less explanatory than `mem_externalized` or `mem-external` or other variants, being made-up will make it easier to grep for and to recognize the connections between these things.

Anything could work, but I propose `refmem` (avoiding "externalized" to avoid confusion with the `@external` annotation, which we are actually _not_ using). So this would be called `testbench_refmem_setup`.